frontend/src/hocs/withMapboxToken/MapboxToken.test.ts

```diff
@@ -18,23 +18,13 @@ import axios from "axios"
 import { SessionInfo } from "src/lib/SessionInfo"
 import AxiosMockAdapter from "axios-mock-adapter"
 import { MapboxToken, TOKENS_URL } from "src/hocs/withMapboxToken/MapboxToken"
+import { mockSessionInfo } from "src/lib/mocks/mocks"
 
-function setSessionInfo(
+function createSessionInfo(
   userMapboxToken = "",
-  commandLine = "streamlit hello"
-): void {
-  SessionInfo.current = new SessionInfo({
-    appId: "aid",
-    sessionId: "mockSessionId",
-    streamlitVersion: "sv",
-    pythonVersion: "pv",
-    installationId: "iid",
-    installationIdV3: "iid3",
-    authorEmail: "ae",
-    maxCachedMessageAge: 2,
-    commandLine,
-    userMapboxToken,
-  })
+  commandLine = ""
```

Comment:
Should this be "streamlit hello"?

Reply:
I removed that because it didn't seem like a sensible default (like, it feels unexpected). Instead, tests that rely on "streamlit hello" being the command line should set it explicitly. I'm pushing a new test to do just this!